modules/ROOT/pages/math.adoc

```diff
@@ -0,0 +1,64 @@
+= Math plugin
+:navtitle: Math Plugin
+:description: {pluginname} plugin provides a way for users to insert a `+<math>+` element into the {productname} editor.
```

Comment:
I wonder if we should not mention that it inserts a math element since technically it inserts a `tiny-math-block` or a `tiny-math-inline` element with a `math` element inside at the moment. But future versions might skip the math element though options if you want to render that your self.

```suggestion
:description: {pluginname} plugin provides a way for users to insert math equation into the {productname} editor.
```